tracery/modifiers.py

```diff
@@ -67,3 +67,130 @@ def lowercase(text, *params):
     'uppercase': uppercase,
     'lowercase': lowercase
 }
+
+
+#----
+#inflect.py modifiers
+#Mappings based on https://github.com/pwdyson/inflect.py
+
+import inflect
+p = inflect.engine()
+
+
+def ordinal(text, *params):
+    return p.ordinal(text)
+
+def number_to_words(text, *params):
+    kwargs=dict(param.split('=') for param in params)
```

Comment:
this function essentially "invents" new syntax for Tracery (key/value parameters to modifiers with `=` as the separator), which I'm not eager to do. I realize that the `number_to_words` function in inflect has a rich set of parameters, but it would make more sense to me in the context of this library to adopt reasonable defaults for these and leave it up to the user to define a more specific modifier function if different functionality is needed.